CMake/Modules/Flow123d_config_header.cmake

```diff
@@ -33,14 +33,18 @@ MACRO (FLOW_DEFINE_CONSTANT variable_name numeric_value)
     MESSAGE (STATUS "Adding: '${variable_name}' ${numeric_value}")
     LIST (APPEND DEFINITIONS_LIST "FLOW123D_${variable_name}=${numeric_value}")
     SET (DEFINITIONS_CONTENT "${DEFINITIONS_CONTENT}#define FLOW123D_${variable_name} ${numeric_value}\n")
-ENDMACRO (FLOW_DEFINE variable_name)
+ENDMACRO (FLOW_DEFINE_CONSTANT variable_name numeric_value)
```

Comment:
Next time, put this into separate commit or pull request as it is an unrelated fix.